src/index.js

```diff
@@ -5,6 +5,7 @@ import './utils/detect_browser';
 import store from './store';
 import App from './App';
 import './index.css';
+import './utils/origin_polyfill';
```

Comment:
@ajvb Please move this to line 5, just below the `detect_browser` import so that this code runs before `App` is imported :+1: then it's all good

Reply:
fixed